listenbrainz/mbid_mapping/README.md

```diff
@@ -0,0 +1,61 @@
+Introduction
+============
+
+This project provides three related components:
+
+1. Artist credit recording mapping which chooses the earliest digital releases from MusicBrainz and then creates
+a table with the first instance of any (artist_credit, recording_name) and maps it to the MBIDs from which it
+originated. This index is useful for resolving ListenBrainz listens where we may only have an artist name and
+a recording name onto sane MusicBrainz release that would be a reasonable choice for what users might expect.
+
+2. The same mapping as above, but focused on having the very first release of any given (artist_credit, recording_name)
+so that it can be used to find the first release year of that pair.
+
+3. A typesense index that takes the data from #1 and generates an index that can be used to quickly and fuzzily
+look up data from the table in #1. This index will serve as the official "MusicBrainz mapping".
+
+Creating mappings and running tests
+-----------------------------------
+
+You will need to have a copy of musicbrainz-docker installed and running in order to run these commands.
+
+
+To build the docker image for the mapping tools:
+
+```./build.sh```
+
+Then to access the manage.py command that is used to invoke the various functions, do:
+
+```docker run --rm -it --network musicbrainzdocker_default metabrainz/mbid-mapping python3 manage.py```
+
+To create the MusicBrainz MBID mapping run:
+
+```manage.py mbid-mapping```
+
+To create the MusicBrainz year mapping run:
+
+```manage.py year-mapping```
+
+
+Creating a typesense index for searching the mapping
+----------------------------------------------------
+
+If you plan to create a typesense index, you'll need typesense installed. First set an API KEY in config.py,
+then run:
+
+```
+docker run -p 8108:8108 -d -v <some_data_dir_for_typesense>/data --name=typesense
```

Comment:
there are now 2 different instructions here for running docker stuff independently. Can this be integrated either into a docker-compose file for mbid-mapping, or into the main LB docker-compose file?

Reply:
I'll leave cleaning this up for the next PR.